sdks/java/io/clickhouse/src/main/java/org/apache/beam/sdk/io/clickhouse/TableSchema.java

```diff
@@ -313,6 +314,10 @@ public static ColumnType parse(String str) {
      * @return value of ClickHouse expression
      */
     public static Object parseDefaultExpression(ColumnType columnType, String value) {
+      if (value == null || value.isEmpty()) {
+        return null;
```

Comment:
while null is fine, return null on value.isEmpty() sounds a breaking change. Previously it returned an empty string on case STRING